File: test/bench/fib/fib.java
class Fib {
    static int fib(int n) {
        if (n < 2) {
            return n;
        }
        return fib(n - 1) + fib(n - 2);
    }
  
    public static void main(String args[]) {
        long start = System.currentTimeMillis();
        int res = fib(30);
        System.out.println("time: " + (System.currentTimeMillis() - start));
        System.out.println(res);
    }
}

File: test/bench/fib/fib.php
<?php  
// jit: `php -d opcache.enable_cli=1 test/bench/fib/fib.php`
$start = hrtime(true);
function fib($n){
    if ($n < 2) {
        return $n;    
    }
    return fib($n-1) + fib($n-2);
}
$res = fib(30);
$time = (hrtime(true) - $start) / 1000000;
echo "time: $time";
echo $res;
?>
